wormhole-connect/src/config/converter.ts

```diff
@@ -167,4 +160,15 @@ export class SDKConverter {
       return this.tokenIdV2(chainName, foreignAddress.toString());
     }
   }
+
+  getNativeTokenAddressV2(token: TokenConfigV1): string | undefined {
+    const address =
```

Comment:
this is minor but I think we overuse ternaries in our code. I'd write it like this:

```ts
if (getGasToken(token.nativeChain).key === token.key && token.key !== 'CELO') {
  return 'native';
}

return token.tokenId?.address
```

idk maybe 5 years of golang rotted my brain but to me this makes it easier to read, and clearer that there's a special case.

Reply:
Will refactor in another PR since I don't want your approval to be dismissed